packages/utxo-lib/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 1.0.4
+
+-   357d195a3 fix(utxo-lib): ltc deserialize without advanced marker
```

Comment:
I am not sure if adding the commit hash is useful since changelog should be for humans, we already have the git log for all the detail history.

Reply:
removed it for now. it felt like a good idea for me initially. 
we should discuss format of changelog while creating an automation script that actually writes the changelog
there is already some degree of automation here https://github.com/trezor/trezor-suite/blob/npm-release/connect/.github/workflows/connect-release-init.yml#L38